cmd/bundle/summary.go

```diff
@@ -32,21 +33,16 @@ func newSummaryCommand() *cobra.Command {
 	cmd.Flags().MarkHidden("include-locations")
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
-		b, diags := prepareBundleForSummary(cmd, forcePull, includeLocations)
 		var err error
+		ctx := logdiag.InitContext(cmd.Context())
+		cmd.SetContext(ctx)
+		logdiag.SetSeverity(ctx, diag.Warning)
 
-		if root.OutputType(cmd) == flags.OutputText {
-			err = render.RenderDiagnostics(cmd.OutOrStdout(), b, diags, render.RenderOptions{RenderSummaryTable: false})
-		} else {
-			err = render.RenderDiagnostics(cmd.OutOrStderr(), b, diags, render.RenderOptions{RenderSummaryTable: false})
```

Comment:
This is a functional change; now the diags are written to stderr and the table to stdout.

This is fine (and even to be expected), but worth calling out. Also, we don't differentiate in our acceptance tests.